src/idx_html.pre.js

```diff
@@ -0,0 +1,76 @@
+const jquery = require('jquery');
+
+/**
+ * The 'pre' function that is executed before the HTML is rendered
+ * @param context The current context of processing pipeline
+ * @param context.content The content
+ */
+function pre(context) {
+  const { document } = context.content;
+  const $ = jquery(document.defaultView);
+
+  const $sections = $(document.body).children('div');
+
+  // first section has a starting image: add title class and wrap all subsequent items inside a div
+  $sections
```

Comment:
maybe it would be better to _run_ the `html.pre.js` here, instead of having duplicate code.

Reply:
good idea! let me refactor it there then